src/mkslides/serve.py

```diff
@@ -53,6 +53,7 @@ def should_watch(
         should_watch(config.slides.theme, REVEALJS_THEMES_LIST),
         should_watch(config.slides.template),
     ]
+    paths_to_watch.extend(Path(path) for path in config.static.values())
```

Comment:
Shouldn't `path` also be relative to `input_path: Path`?

Reply:
Same as above, paths are already `cwd`-relatives and can be given to watcher as they are as they share the same `cwd`.
It has been working without needing extra transformation